strings/names/death_commando.txt

```diff
@@ -61,7 +61,7 @@ Splint Chesthair
 Stabby McGee
 Stump Beefgnaw
 Stump Chunkman
-THAT DAMN FAGGOT TRAITOR GEORGE MELONS
+THAT DAMN meatball TRAITOR GEORGE MELONS

```

Comment:
This is more funny then before

Reply:
Thank you for appreciating my replacing humor.